source/content/guides/git/06-resolve-merge-conflicts.md

```diff
@@ -0,0 +1,186 @@
+
+This is safe to run if you don't have your own changes in any of the conflicting files (for example problems with `.gitignore`).
+
+<TabList>
+
+<Tab title="Drupal 7" id="d7" active={true}>
+
+  ```bash{promptUser: user}
+  git pull -Xtheirs https://github.com/pantheon-systems/drops-7.git master
+  # resolve conflicts
+  git push origin master
+  ```
+
+</Tab>
+
+<Tab title="WordPress" id="wp">
+
+  ```bash{promptUser: user}
+  git pull -Xtheirs https://github.com/pantheon-systems/WordPress.git master
+  # resolve conflicts
+  git push origin master
+  ```
+
+</Tab>
+
+<Tab title="WordPress Site Network" id="wp-network">
+
+  ```bash{promptUser: user}
+  git pull -Xtheirs https://github.com/pantheon-systems/wordpress-network.git master
+  # resolve conflicts
+  git push origin master
+  ```
+
+</Tab>
+
+</TabList>
+
+1. Double-check the files before going forward to make sure no bugs were introduced.
+
+    - The `-Xtheirs` flag will drop your changes if you modify core CMS files.
+    You should [manually resolve conflicts](#manually-resolve-conflicts) in this scenario. 
+  
+1. Review [WordPress and Drupal Core Updates](/core-updates) for more details on core updates.
+
+### Find a Site's Upstream URL
+
+1. Run the following command in [Terminus](/terminus) to retrieve your Upstream URL:
+
+   ```bash{promptUser: user}
+   terminus site:info <site> --field=upstream
+   ```
+
+## Manually Resolve Conflicts
+
+Steps to resolve merge conflicts vary by the type of conflict. Refer to the sections below for more information on resolving delete/modify conflicts, content conflicts, and Multidev conflicts.
+
+### Resolve Delete/Modify Conflicts
+
+A delete/modify conflict occurs when one commit deletes a file and another modifies it. Follow the steps below to resolve such a conflict.
+
+
+1. Identify the file that is generating a delete error.
+
+    For example, the Git log may contain an entry similar to the following:
+
+    ```git
+    CONFLICT (delete/modify): scripts/run-tests.sh deleted in HEAD and modified in 72faeeff1c9356221694d1351cdb2000ab3c5d1c. Version 72faeeff1c9356221694d1351cdb2000ab3c5d1c of scripts/run-tests.sh left in tree.
+    ```
+
+1. Navigate to your local repository > run the following Git command to get a copy of the file in conflict:
+
+    ```bash{promptUser: user}
+    git checkout <commit ID> -- <file>
+    ```
+
+    <Alert title="Note" type="info">
+
+    When looking for a commit ID, you can find the last instance where the missing file was in the repository.
+
+    </Alert>
+
+1. Run `git status` and verify that there is a new file to add to the repository:
+
+ ```bash{outputLines:2-5}
+  git status
+  On branch master
+  Changes to be committed:
+  (use "git reset HEAD ..." to unstage)
+  new file: README.txt
+  ```
+
+1. Stage and commit:
+
+ ```bash{promptUser: user}
+ git commit -am "verifying missing README.txt"
+ ```
+
+  You will receive confirmation from Git that the file has been committed.
```

Comment:
Is this always true?  Isn't it possible for folks to turn off all notifications?  Or does the confirmation appear in the terminal?

Reply:
It appears in the terminal.